### ISSUE TITLE
crop in transformed coordinates

### ISSUE THREAD

**Issue by martinschorb** [bigdataprocessor/bigdataprocessor2#168]
Hi,

I like to crop a dataset in an arbitrary orientation.

However, the bounding box is only shown in the original volume coordinates. Can I provide a transformed source and crop it in the viewer's frame?

**Comment by tischi** [comment]
Yes, you can using: [Process > Transform] (enter an affine transform) and then _afterwards_ do [Process > Crop...]. Would that work for you?

**Comment by tischi** [comment]
I can also try to make the Manual Transform `[T]` work (which currently does not); would that help?

**Comment by martinschorb** [comment]
My actual intention was to use the cropping to define sources for a gallery view of subvolumes. https://github.com/mobie/mobie.github.io/issues/32
I am trying to find out how the cropping and transformation could be best stored in a MoBIE2 compatible way.

**Comment by martinschorb** [comment]
so it seems there is no way of storing a crop only in metadata. Even when I load a BDV/XML dataset into BDP2 and crop it saving requires to write a new data container. Would it be possible to do this entirely at the metadata level? So that instead of defining transformations only, I could also define the crop bounding box? Ideally in the same logic of a transformation... So one could even do:
```
transformation -> crop (arbitrary BBox orientation) -> transform (display)
```

**Comment by constantinpape** [comment]
> @constantinpape @K-Meech do you feel having a spec for a crop of an image in MoBIE would be something generally useful (to me it feels that could be something useful in order show sub-volumes of big image data sets)

Yes, I agree that could be useful, as already discussed in https://github.com/mobie/mobie.github.io/issues/32.
I have made an issue in the mobie repo on this https://github.com/mobie/mobie.github.io/issues/44.

**Comment by martinschorb** [comment]
> Would you already have a convenient workflow for how to determine the transformation and crop

Yes, that is all ready to go. The scenario is: 

- volume that includes a centriole
- I transform such that the major axis of the centriole is in the center of the FOV and parallel to `x`.
- now I want to only show a gallery of many centrioles in that orientation
- this means I need to crop the individual sources to the subvolume around the centrioles
- so the BBox would be `center +- xyz` in physical coordinates after applying this targeting transformation
